packages/perseus-editor/src/widgets/interactive-graph-editor/components/interactive-graph-description.tsx

```diff
@@ -52,23 +56,21 @@ export default function InteractiveGraphDescription(props: Props) {
                         />
                     </LabelLarge>
                     <Strut size={spacing.small_12} />
-                    <LabelLarge tag="label">
+                    <LabelLarge tag="label" htmlFor={descriptionTextAreaId}>
                         Description
-                        <TextArea
-                            rows={8}
-                            resizeType="vertical"
-                            value={ariaDescriptionValue}
-                            onChange={(newValue) =>
-                                onChange({
-                                    fullGraphAriaDescription:
-                                        // Save as undefined if the new value
-                                        // is an empty string.
-                                        newValue || undefined,
-                                })
-                            }
-                            style={styles.spaceAbove}
-                        />
```

Comment:
It won't let me use the textarea as the child of a label anymore after changing it to a non-wonder blocks component.